desc/compute/_stability.py

```diff
@@ -200,3 +200,165 @@ def _magnetic_well(params, transforms, profiles, data, **kwargs):
         / (data["V_r(r)"] * data["<B^2>"])
     )
     return data
+
+
+@register_compute_fun(
+        name="gds2",
+        label="\\mathrm{gds2}",
+        units="~",
+        units_long="None",
+        description="Fieldline bending term1",
+        dim=3,
+        params=[],
+        transforms={"grid": []},
+        profiles=[],
+        coordinates="rtz",
+        data=[
+            "grad(alpha)",
+            ]
+        )
+def _gds2(params, transforms, profiles, data, **kwargs):
+    data["gds2"]= dot(data["grad(alpha)"], data["grad(alpha)"])
+
+
+@register_compute_fun(
+        name="gds21",
+        label="\\mathrm{gds21}",
+        units="~",
+        units_long="None",
+        description="Fieldline bending term2",
+        dim=3
+        params=[]
+        transforms={"grid": []},
+        profiles=[],
+        coordinates="rtz",
+        data=[
+            "grad(alpha)",
+            "grad(rho)"
+            ]
+        )
+def _gds21(params, transforms, profiles, data, **kwargs):
+    data["gds21"]= dot(data["grad(alpha)"], data["grad(rho)"])
+
+
+@register_compute_fun(
+        name="gds22",
+        label="\\mathrm{gds22}",
+        units="~",
+        units_long="None",
+        description="Fieldline bending term3",
+        dim=3
+        params=[]
+        transforms={"grid": []},
+        profiles=[],
+        coordinates="rtz",
+        data=[
+            "grad(rho)"
+            ]
+        )
+def _gds21(params, transforms, profiles, data, **kwargs):
+    data["gds22"]= dot(data["grad(rho)"], data["grad(rho)"])
+
+
+@register_compute_fun(
+        name="cvdrift",
+        label="\\mathrm{curvature-drift}",
+        units="~",
+        units_long="None",
+        description="Geometric part of the curvature drift"+"used for local stability analyses",
+        dim=3
+        params=[]
+        transforms={"grid": []},
+        profiles=[],
+        coordinates="rtz",
+        data=[
+            "dp_dpsi",
+            "psi_r",
+            "B",
```

Comment:
this is the vector B, is that what you want? You might want `|B|^2`

Reply:
Yes, I want |B| here.